build/dev-env.sh

```diff
@@ -36,7 +36,7 @@ if ! command -v kind &> /dev/null; then
 fi
 
 if ! command -v kubectl &> /dev/null; then
-  echo "Please install kubectl 1.15 or higher"
+  echo "Please install kubectl 1.19 or higher"
```

Comment:
if you require >1.19 then probably want to check the output of kubectl version instead (like helm)